lib/game_parser.rb

```diff
@@ -7,4 +9,12 @@ def initialize(file)
   def get_first_line
     File.open(@file, &:readline)
   end
+
+  def get_log_metadata
+    path = File.expand_path(@file)
+    lines = File.readlines(@file).length
+    metadata = {path => {"lines" => lines}}
+    json = JSON.pretty_generate(metadata)
+    json[1..json.length-2]
```

Comment:
Pode explicar o que você ta fazendo aqui?

Reply:
Capturando o número de linhas e montando um hash para convertê-lo em JSON. Então, retorno o JSON sem as chaves do início e do fim, como especificado no desafio:
```
"games.log": {
  "lines": 0
}
```